src/Core/Services/Implementations/CipherService.cs

```diff
@@ -790,5 +799,18 @@ private async Task<bool> UserCanEditAsync(Cipher cipher, Guid userId)
 
             return await _cipherRepository.GetCanEditByIdAsync(userId, cipher.Id);
         }
+
+        private async Task ValidateCipherLastKnownRevisionDateAsync(Cipher cipher, DateTime? lastKnownRevisionDate)
+        {
+            if (cipher.Id == default || !lastKnownRevisionDate.HasValue) return;
```

Comment:
Please add the `return;` line inside of a curly brace block; we use that for all control statements/blocks, even 1-liners.
```c#
if (...)
{
    return;
}
```